src/main/app/src/constants/keys.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const info = [{
```

Comment:
We are going to create a servlet to retrieve the client id but for now this works.

Reply:
Awesome!